service/ui_plugin_build_baron.go

```diff
@@ -120,6 +122,10 @@ func (uis *UIServer) bbJiraSearch(rw http.ResponseWriter, r *http.Request) {
 		return
 	}
 	if err != nil {
+		grip.Debug(message.Fields{
```

Comment:
is this one necessary? It looked to me like we've already logged every way GetSearchReturnInfo could return an error.

Reply:
true, I'll remove it